app/processing/invoice-line/get-invoice-line-by-funding-code-and-payment-id.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../../data')
+
+const getInvoiceLineByFundingCodeAndPaymentId = async (fundingCode, paymentRequestId) => {

```

Comment:
getPositiveNonZeroInvoiceLineByFundingCodeAndPaymentRequestId

Reply:
Also, should this be part of the transaction?

Reply:
Value may be zero, hence I changed name to get getPositiveInvoiceLineByFundingCodeAndPaymentId